package.json

```diff
@@ -41,6 +42,7 @@
     "devDependencies": {
         "autoprefixer": "6.3.6",
         "cssnano": "3.7.1",
+        "gscan": "^2.0.0",
```

Comment:
@kirrg001 what do you think of pinning gscan like this - so it will install 2.0.1 and even 2.1.0 as we go through the versions? 

Casper should always be up to date :)

Reply:
🤘🏽